src/rpc/rawtransaction.cpp

```diff
@@ -2689,13 +2689,13 @@ static RPCHelpMan rawreissueasset()
             throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid asset address provided: %s", asset_address_uni.get_str()));
         }
 
-        size_t input_index = -1;
+        int input_index = -1;
```

Comment:
May as well use the fix from dgpv's old comment here, and push the declaration of input_index inside the conditional so we don't need to separately initialize it at all.

Reply:
I think this fix matches what Core does.

Reply:
(If you want to leave it as-is, that's also fine.)